Cargo.toml

```diff
@@ -37,6 +37,7 @@ wayland-server = { version = "0.28.3", optional = true }
 wayland-sys = { version = "0.28", optional = true }
 winit = { version = "0.24.0", optional = true }
 xkbcommon = "0.4.0"
+scan_fmt = "0.2"
```

Comment:
Since we do not need it, could you disable the regex feature of scan_fmt?

Reply:
> Small nitpick in an effort to keep the list of our dependencies (relatively) small.

This was one of my concers too, i tried to avoid it, but string parsing is not that trivial and i tried to find a small lib that can
free me from that burden :)

> Since we do not need it, could you disable the regex feature of scan_fmt?

I will check it out and update the PR

> (And reminds me, I really need to do a rustfmt run against this PR. @cmeissl Would you prefer me to format the code before or after #272? If I format this before, I would prefer a rebase on your PR.)

You can format the code before #272 if you like to, no problem.